src/DynamicRupture/Output/RateAndState.hpp

```diff
@@ -7,11 +7,11 @@ namespace seissol::dr::output {
 class RateAndState : public Base {
   public:
   void tiePointers(seissol::initializers::Layer& layerData,
-                   seissol::initializers::DynamicRupture* dynRup,
+                   seissol::initializers::DynamicRupture* drDescr,
```

Comment:
It may so happen that `DynamicRupture` was not a good class name. Description is better in my opinion because it defines concrete names of the variables in an LTS tree. If we decide to change the class name then I would suggest to wait till the merge with the master

Reply:
ok